pkg/builder/local_build_executor.go

```diff
@@ -240,8 +271,12 @@ func (be *localBuildExecutor) Execute(ctx context.Context, filePool re_filesyste
 	// Build rules generally expect the parent directories to already be
 	// there. We later use the directory handles to extract output files.
 	outputParentDirectories := map[string]filesystem.Directory{}
+	workingDirectory := command.WorkingDirectory
+	if workingDirectory == "" {
```

Comment:
I just tested: there is likely no need to set `workingDirectory` in case it's empty. `path.Join()` does the right thing when the first argument is the empty string.

Reply:
Good point, that's what `path.Clean()` gives you. Thanks.